docs/testing.rst

```diff
@@ -27,34 +27,34 @@ Example::
     import click
     from click.testing import CliRunner
 
-    def test_hello_world():
-        @click.command()
-        @click.argument('name')
-        def hello(name):
-            click.echo('Hello %s!' % name)
+    @click.command()
+    @click.argument('name')
+    def hello(name):
+        click.echo('Hello %s!' % name)
 
+    def test_hello_world():
         runner = CliRunner()
         result = runner.invoke(hello, ['Peter'])
         assert result.exit_code == 0
         assert result.output == 'Hello Peter!\n'
-
+        
```

Comment:
@dendrondal whitespace?

Reply:
Thank you! Not sure why it didn't copy over properly.